src/P5/Image/LoadingAndDisplaying.purs

```diff
@@ -42,8 +47,6 @@ image2 p5 img dx dy dWidth dHeight sx sy sWidth sHeight = runFn10 image2Impl p5
 imageMode :: P5 -> ImageMode -> (Effect Unit)
 imageMode p5 mode = runFn2 imageModeImpl p5 mode
 
--- TODO: unsupported: loadImage :: P5 -> String -> Unsupported(function(p5.Image)) -> Unsupported(Function(Event)) -> Image
```

Comment:
I don't quite understand that `Unsupported(function(p5.Image)) ` part to be honest...

Reply:
It just documents the types in the p5.js javadoc that don't have corresponding types in P5.Types.purs.